src/utils/authorization/jwt.go

```diff
@@ -1,2 +1,25 @@
 package authorization
 
+import (
+	"github.com/dgrijalva/jwt-go"
+	"time"
+)
+
+var jwtKey = []byte("secret_key")
+
+type Token struct {
+	UserName  string    `json:"username"`
+	ExpiresAt time.Time `json:"expires_at"`
+}
```

Comment:
Keep model in genericModel as mentioned earlier

Reply:
Done